pygfx/resources/_texture.py

```diff
@@ -253,10 +265,33 @@ def usage(self):
     @property
     def colorspace(self):
         """If this data is used as color, it is interpreted to be in this colorspace.
-        Can be "srgb" or "physical". Default "srgb".
+        Can be "srgb", "physical", "yuv420p", or "yuv444p". Default "srgb".
+
+        * "srgb": the data represents intensity, rgb, or rgba pixels in the sRGB space.
+          sRGB is a standard color space designed for consistent representation of colors
+          across devices like monitors. Most images store colors in this space.
+        * "physical": the colors are in the physical / linear space, where lighting
+          calculations can be applied. The shader convers sRGB colors to physical in
+          the shader before doing color computations.
+        * "yuv420p": A common video format. The data is represented as 3 planes (y, u, and v).
+          The y represents intensity, and is at full resolution. The u and v planes are a
+          quarter of the size. The planes must be stored in two layers of the texture,
+          with the u and v plane next to each-other in top half the second layer.
+        * "yuv444p": A lesser common video format. The data is represented as 3 planes
+          (y, u, and v) similar to yuv420p however the u and v planes are stored
+          at full resolution.
```

Comment:
Nice to have support for yuv444p too, makes a lot of sense, it being a simpler variant of yuv420p.

Reply:
yeah, i wanted to mostly compare the performance of difference color conversions.